packages/components/src/alignment-matrix-control/index.tsx

```diff
@@ -30,7 +35,27 @@ function useBaseId( id ) {
 	return id || instanceId;
 }
 
-export default function AlignmentMatrixControl( {
+/**
+ *
+ * AlignmentMatrixControl components enable adjustments to horizontal and vertical alignments for UI.
+ *
+ * ```jsx
+ * import { __experimentalAlignmentMatrixControl as AlignmentMatrixControl } from '@wordpress/components';
+ * import { useState } from '@wordpress/element';
+ *
+ * const Example = () => {
+ * 	const [ alignment, setAlignment ] = useState( 'center center' );
+ *
+ * 	return (
+ * 		<AlignmentMatrixControl
+ * 			value={ alignment }
+ * 			onChange={ ( newAlignment ) => setAlignment( newAlignment ) }
```

Comment:
Nit: We shouldn't have to create a new function here

```suggestion
 * 			onChange={ setAlignment }
```